.gitmodules

```diff
@@ -20,3 +20,9 @@
 [submodule "examples/rust"]
 	path = examples/rust
 	url = https://github.com/songtianyi/tdengine-rust-bindings.git
+[submodule "src/connector/php-tdengine"]
+	path = src/connector/php-tdengine
+	url = https://github.com/Yurunsoft/php-tdengine.git
+[submodule "src/connector/tdengine-restful-connector"]
```

Comment:
I think tdengine-restful-connector is a general name that might confuse the user. I suggest using a name with 'php' to be more indicative like another one you submitted.